app/frontend_messages.py

```diff
@@ -0,0 +1,15 @@
+from flask.ext.babel import gettext as _
+
+
+MESSAGES = {
+            'multiple-surveys': {
+                'TITLE': _("Information"),
+                'MSG': _("Unfortunately you can only complete one survey at a time."),
```

Comment:
I was under the impression the copy for this message had been changed? Why's it unfortunate?